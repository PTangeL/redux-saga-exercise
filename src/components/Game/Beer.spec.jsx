import React from 'react'
import { shallow, mount } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Beer from './Beer'

describe('<Beer />', () => {
  const props = {
    beer: {},
    setLoadingBeer: () => {},
    loadingBeer: false
  }

  const wrapper = shallow(
    <Beer {...props} />
  )

  it('should render with props', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })

  it('Method #componentDidMount() should assign beerImgWrapper.style.minHeight', () => {
    //TODO implement test
  })

  it('Method #imageLoaded() should...', () => {
    //TODO implement test
  })

  it('Method #imageLoaded() should...', () => {
    //TODO implement test
  })

})
