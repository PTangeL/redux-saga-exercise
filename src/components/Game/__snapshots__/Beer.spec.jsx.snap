// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Beer /> should render with props 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-4"
      />
      <div
        className="col-sm-4 text-center beer-col"
      >
        <div
          className="beer-img"
        >
          <img
            className="img-responsive"
            height="autopx"
            name="beerImg"
            onError={[Function]}
            onLoad={[Function]}
            role="presentation"
            style={
              Object {
                "color": "red",
                "height": "auto",
              }
            }
          />
        </div>
        <h3
          className="beer_title"
          name="beerTitle"
        />
      </div>
      <div
        className="col-sm-4 hidden-xs hidden-sm"
      >
        <BeerAttributes
          beer={Object {}}
        />
      </div>
    </div>
  </div>
</div>
`;
